src/font/bitmapfont.js

```diff
@@ -91,32 +95,49 @@
          * @param {Number} scale ratio
          */
         resize : function (scale) {
-            // updated scaled Size
-            this.sSize.setV(this.fontSize);
-            this.sSize.x *= scale;
-            this.sSize.y *= scale;
-            this.height = this.sSize.y;
+            this.fontScale.set(scale, scale);
+        },
+
+        /**
+         * Measures a single line of text, does not account for \n
+         * @name measureTextLineWidth
+         * @memberOf me.BitmapFont
+         * @function
+         * @param {String} text
+         * @returns {Number} the calculated width
+         */
+        measureTextWidth : function(text) {
```

Comment:
not sure we need this one as a public function, maybe I'd personally put it as a global (utility) function under `me.BitmapFont` and then pass the font data as extra argument.